docs/integrations/ios-specific-workflow-steps/xcode-select.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Xcode Select
+metaTitle: Xcode Select
+metaDescription: Xcode Select
+sidebar_position: 1
+---
+import Screenshot from '@site/src/components/Screenshot';
+
+# Xcode Select (Version)
+
+This step is used to specify the Xcode version to be used during the build process. All available versions of Xcode can be seen in the [Configuration](https://docs.appcircle.io/build/build-profile-configuration) tab. 
+
+### Prerequisites
+
+Always use this step **before** **CocoaPods Install** and **Xcodebuild for Devices**. If you have other **Xcode related** steps, such as **Xcodebuild for iOS Simulators** and **Xcodebuild for Unit and UI Tests**, **don't forget** to use before them. 
```

Comment:
Here you said that these two steps:
- CocoaPods Install
- Xcodebuild for Devices
should be used before this step. However, you added it  in the Prerequisites table below. 
Can you please fix the logical error here?

If it needs to be used **afterwards**, there should be a warning, but it should **not be in the Prerequisites table**.